api/requirements_dev.in

```diff
@@ -10,7 +10,7 @@ pip-tools
 
 # Useful extra developer packages:
 django-debug-toolbar
-flake8
+flake8<4
```

Comment:
Bij dependency pinning doen we altijd een comment erbij waarom. Dan begrijpen onze opvolgers ook weer waarom dat was.

Reply:
Goede - ik zet er een comment bij